src/webpack/create-controllers-module.js

```diff
@@ -39,16 +46,26 @@ module.exports = function createControllersVirtualModule(config) {
             const controllerMain = packageName + '/' + controllerPackageConfig.main;
             const webpackMode = controllerUserConfig.webpackMode;
 
-            file +=
+            controllerContents +=
                 "\n  '" +
                 controllerReference +
                 '\': import(/* webpackMode: "' +
                 webpackMode +
                 '" */ \'' +
                 controllerMain +
                 "'),";
+
+            if ('undefined' !== typeof controllerUserConfig.autoimport) {
+                for (let autoimport in controllerUserConfig.autoimport) {
```

Comment:
Wouldn't using `controllerUserConfig.autoimport || []` work here instead of the if?

Reply:
I'm copying your code Tito! ;)